docs/topics/conditional-view-processing.txt

```diff
@@ -198,6 +201,14 @@ to compute the ETag and last modification values in all situations. In fact,
 you **should** use the same functions, so that the same values are returned
 every time.
 
+.. note::
```

Comment:
I typically prefer to use `.. admonition:: descriptive title` rather than "note" to improve skimmability.